app/views/groups/index.html.erb

```diff
@@ -34,6 +34,9 @@
   <div class="container text-center mt-5 py-3">
     <h3>
       <%= group.name %>
+      <% if group.verified?%>
+        <i class="bi bi-patch-check"></i>
+      <% end %>
```

Comment:
Should be moved to a template which is then rendered instead of `group.name`

Reply:
c96c359 should resolve this. If you agree, please resolve this conversation.

Reply:
Rendering is now moved to a template. Will resolve although using a partial was not accepted because discussion about using ViewComponent instead is tracked [here](https://github.com/hpi-swt2/bookkeeper-portal-red/pull/192#discussion_r1064171527).